src/renderer/widgets/CreateWallet/ui/MultisigWallet/components/Signer.tsx

```diff
@@ -0,0 +1,49 @@
+import { BN_ZERO } from '@polkadot/util';
+import { useUnit } from 'effector-react';
+import { type FormEvent } from 'react';
+
+import { type AccountId, type Chain } from '@/shared/core';
+import { toAddress, transferableAmount } from '@/shared/lib/utils';
+import { Icon } from '@/shared/ui';
+import { AssetBalance } from '@/entities/asset';
+import { balanceModel, balanceUtils } from '@/entities/balance';
+import { AddressWithName } from '@/entities/wallet';
+
+interface Props {
+  onSubmit: (event: FormEvent, accountId: AccountId) => void;
+  accountId: AccountId;
+  name?: string;
+  chain: Chain;
+}
+
+export const Signer = ({ accountId, name, onSubmit, chain }: Props) => {
+  //   const balance = useBalance({
```

Comment:
I have an issue here, where only the balance of the selected account is displayed. If I have 2 possible signers, and I want to display the balance of each of them, the balance is `undefined` for the account that isn't selected.

I tried 2 approaches, one using `useBalance`, the other one fetching the balance directly from `balanceModel`. Both ways failed in the same way.

Can you help @johnthecat @sokolova-an ?

Here is an example, I own both accounts, they both have > 0 WND but only the one that is selected (in blue) has a balance.
![image](https://github.com/user-attachments/assets/3ddae3a6-aff7-47fa-ac53-805fffacd584)

Reply:
That's a tricky case, as I remember we don't load balances for all wallets at once, so if you haven't ever opened the wallet - its accounts won't be in the balances storage and you can't get access to the value with balanceModel in this case.
Please check, will it work if before that, you open the second wallet and wait until the balances are loaded?

In any case, I guess, you will have to manually run update or populate balances for the other wallet, because the store data can be outdated anyway. The main logic about balance stores in `balanceSubModel`

Reply:
Thank you so much for the pointers. Even when I had the other account open before the balance was undefined. I subscribed manually to the balance now, and this fixed it. Thanks again.